lib/tooltip.css

```diff
@@ -3,6 +3,7 @@
     padding: 5px;
     border: 1px solid sandybrown;
     border-radius: 5px;
+    z-index: 1000;
```

Comment:
Why 1000 ? How do you know there is no element with z-index > 1000 in other projects ? You cannot add a rule in your project's css to handle this case ?